src/gateway/api/V1/serializers/hot_water.py

```diff
@@ -0,0 +1,51 @@
+
+from gateway.dto.hot_water import HotWaterDTO, HotWaterStatusDTO
+
+if False:  # MYPY
+    from typing import Any, Dict, List, Optional
+
+class HotWaterConfigSerializer(object):
+
+    @staticmethod
+    def serialize(hot_water_dto):
+        # type: (HotWaterDTO) -> Dict[str,Any]
+        data = {'id': hot_water_dto.id,
+                'external_id': hot_water_dto.external_id,
+                'name': hot_water_dto.name,
+                }
+        if hot_water_dto.plugin:
+            data['plugin'] = hot_water_dto.plugin.name
+        if hot_water_dto.source:
+            data['source'] = hot_water_dto.source.source
+        if hot_water_dto.status:
+            data['status'] = hot_water_dto.status
```

Comment:
```suggestion
            data['status'] = HotWaterStatusSerializer.serialize(hot_water_dto.status)
```

This is a bit nicer than having to serialize explicitly in each api call.